chapters/4.md

```diff
@@ -130,7 +130,7 @@ When Monero was launched, the coin emission rate was more than 30 XMR every 2 mi
 
 Many cryptocurrencies have a fixed cap on coin emission, with a hard limit on maximum supply. Once the supply cap is reached, there will be no new coins introduced for miners, who will be forced to subsist entirely on fees. For example, in Bitcoin, this paradigm shift in financial incentives will occur around 2140, when the supply reaches 21 million Bitcoins and ceases to increase any further. This approach is often touted as a benefit to remain “deflationary,” however these arguments are often based on conflating the concept of an inflating monetary supply with a different use of the word “inflation” to describe an undesirable decrease in spending power of a currency.
 
-The annual supply increase from Monero's 0.6 XMR tail emission is less than 1% per year. While developers created tail emission to incentivize miners long-term, a neat by product is its easy comparison to to asset backed financial systems. The tail emission rate of supply increase is comparable to the annual gold supply increase globally, which in modern civilization has historically been the purest and most stable form of money. Since the miners will always be able to collect fees, the guaranteed coinbase payments provide greater financial stability for the miners in the long run. This social contract ensures that the miners remain incentivized to use their equipment to secure the Monero network.
+The annual supply increase from Monero's 0.6 XMR tail emission is less than 1% per year. While developers created tail emission to incentivize miners long-term, a neat by-product is its easy comparison to asset backed financial systems. The tail emission rate of supply increase is comparable to the annual gold supply increase globally, which in modern civilization has historically been the purest and most stable form of money. Since the miners will always be able to collect fees, the guaranteed coinbase payments provide greater financial stability for the miners in the long run. This social contract ensures that the miners remain incentivized to use their equipment to secure the Monero network.
```

Comment:
Here I think saying "annual" and "per year" in the same sentence looks redundant.
My suggestion would be to rephrase it:

`The annual supply increase from Monero's 0.6 XMR tail emission is less than 1%.`

or

`The supply increase from Monero's 0.6 XMR tail emission is less than 1% per year.`

Reply:
```The supply increase from Monero's 0.6 XMR tail emission is less than 1% per year.``` seems a good choice